tensorboard/plugins/metrics/metrics_plugin.py

```diff
@@ -305,10 +306,16 @@ def is_active(self):
     def _serve_tags(self, request):
         ctx = plugin_util.context(request.environ)
         experiment = plugin_util.experiment_id(request.environ)
-        index = self._tags_impl(ctx, experiment=experiment)
+
+        run_ids = None
+        query_params = urllib.parse.parse_qs(request.environ["QUERY_STRING"])
+        if query_params.get("runIds"):
+            run_ids = query_params.get("runIds")
+
+        index = self._tags_impl(ctx, experiment=experiment, run_ids=run_ids)
         return http_util.Respond(request, index, "application/json")
 
-    def _tags_impl(self, ctx, experiment=None):
+    def _tags_impl(self, ctx, experiment=None, run_ids=None):
```

Comment:
Let's call this `runs` not `run_ids`. DataProvider's [`Run` object](https://cs.opensource.google/tensorflow/tensorboard/+/master:tensorboard/data/provider.py;l=439;drc=4b5ce965b0c2f8ee22d6c9c1e4842b5e8fd7b9f4) technically distinguishes between the name and the ID. In practice, they're often interchangeable, but the existing DataProvider APIs are built to expect run names, including the RunTagFilter object.

So since we are passing this value back into DataProvider via RunTagFilter, we should just pass the run name, not the ID.

Ditto for the plumbing elsewhere.

Reply:
That's good to know, I'll rename everything to `runs` and indicate these should be run names in the doc.